examples/lights/Makefile

```diff
@@ -23,7 +23,8 @@
 TARGET := lights
 FX2LIBDIR := ../../third_party/fx2lib
 GITMODULESDIR := ../../.gitmodules
+BOARD ?= fx2
```

Comment:
I think we need a better name for the FX2 dev board you are using rather than just "FX2" -- there is also the official FX2 dev board that Cypress released.

Maybe `fx2miniboard` or something like that?